dashboard/features/layout/navigation.tsx

```
@@ -0,0 +1,114 @@
+import { BotIcon, CircleGaugeIcon, CreditCardIcon, HeadsetIcon, WebhookIcon } from "lucide-react";
```

ESLint

[error] Unable to resolve path to module 'lucide-react'.

(import-x/no-unresolved)